projects/cn0531/pinmux_config.c

```diff
@@ -0,0 +1,48 @@
+/*
+ **
+ ** Source file generated on November 18, 2019 at 09:16:12.
+ **
+ ** Copyright (C) 2011-2019 Analog Devices Inc., All Rights Reserved.
+ **
+ ** This file is generated automatically based upon the options selected in
+ ** the Pin Multiplexing configuration editor. Changes to the Pin Multiplexing
+ ** configuration should be made by changing the appropriate options rather
+ ** than editing this file.
+ **
+ ** Selected Peripherals
+ ** --------------------
+ ** SPI1 (SCLK, MISO, MOSI, CS_0)
+ ** UART0 (Tx, Rx)
+ **
+ ** GPIO (unavailable)
+ ** ------------------
+ ** P0_10, P0_11, P1_06, P1_07, P1_08, P1_09
+ */
+
+#include <sys/platform.h>
+#include <stdint.h>
+
+#define SPI1_SCLK_PORTP1_MUX  ((uint16_t) ((uint16_t) 1<<12))
```

Comment:
Maybe use BIT() macro

Reply:
This is a autogenerated file for aducm3029